ui/src/components/Common/RadioButtonFieldSet.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+<div v-for="category of radioValues" :key="category.key" class="p-field-radiobutton">
+    <RadioButton :id="category.key" :name="name" :value="category.name" v-model="selectedVal" @change="onSelect"/>
+    <label :for="category.key">{{category.name}}</label>
+</div>
+</template> 
+
+<script setup lang="ts">
+
+import {defineEmits, ref, defineProps } from "vue";
```

Comment:
`defineProps` and `defineEmits` do not need to be imported and are compiled away when <script setup> is processed.
Looks like these imports can also be removed from other components.

Reply:
Changes are done